Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,110 @@
+/* Joe Johnson Style Sheet */
+
+body {
+	width: 700px; 
+	margin: 20px auto;
+	padding-left: 30px;
+    padding-right: 30px;
+	border: 5px solid black;
+	border-top-style: solid;
+    border-left-style: none; 
+    border-right-style: none;
+    border-bottom-style: solid;
+}
+
+h1, 
+h2, 
+h3, 
+h4, 
+h5, 
+h6 {
+	font-weight: bold;
+	font-family: sans-serif;
+}
+
+h1, h4 {
+	text-transform: uppercase;
+}
+
+h1, h2 {
+	margin-bottom: 30px
+}
+
+h1 {
+	margin-top: 5px;
+}
+
+h2 {
+	margin-top: 30px;
+}
+
+h4 {
+	margin-top: 40px;
+	margin-bottom: 20px;
+	letter-spacing: 2px;
+}
+
+p {
+	margin-top: 0px;
+	line-height: 25px;
+	font-size: 18.5px;
```

Comment:
Super minor, but it's not really reliable to make measurements in something less than a full pixel, so avoid setting pixel measurements with any decimal places.